src/Gameplay/Foundations.java

```diff
@@ -0,0 +1,69 @@
+package Gameplay;
+
+import Cards.Card;
+
+import java.awt.*;
+import java.util.Objects;
+
+public class Foundations extends Pile {
+
+    private int suit;
```

Comment:
Changes to suits might be odd.

```suggestion
    private final int suit;
```